src/components/moderation/ReportDialog/utils/useReportOptions.ts

```diff
@@ -1,121 +1,321 @@
+          title: _(msg`Unlabelled GenAI or Synthetic Content`),
+          reason: 'tools.ozone.report.defs#reasonMisleadingSyntheticContent',
+        },
+        {
+          title: _(msg`Harmful False Claims`),
+          reason: 'tools.ozone.report.defs#reasonMisleadingMisinformation',
         },
         {
-          reason: ComAtprotoModerationDefs.REASONSEXUAL,
-          title: _(msg`Unwanted Sexual Content`),
-          description: _(msg`Inappropriate messages or explicit links`),
+          title: _(msg`Other Misleading Content`),
+          reason: 'tools.ozone.report.defs#reasonMisleadingOther',
         },
-        ...common,
       ],
-      list: [
+    },
+    ruleBreaking: {
+      key: 'ruleBreaking',
+      title: _(msg`Breaking Network Rules`),
+      description: _(msg`Hacking, stolen content or prohibited sales`),
+      sort: 6,
+      options: [
         {
-          reason: ComAtprotoModerationDefs.REASONVIOLATION,
-          title: _(msg`Name or Description Violates Community Standards`),
-          description: _(msg`Terms used violate community standards`),
+          title: _(msg`Hacking or System Attacks`),
+          reason: 'tools.ozone.report.defs#reasonRuleSiteSecurity',
+        },
+        {
+          title: _(msg`Stolen Content`),
+          reason: 'tools.ozone.report.defs#reasonRuleStolenContent',
+        },
+        {
+          title: _(msg`Promoting or Selling Prohibited Items of Services`),
+          reason: 'tools.ozone.report.defs#reasonRuleProhibitedSales',
+        },
+        {
+          title: _(msg`Banned User Returning`),
+          reason: 'tools.ozone.report.defs#reasonRuleBanEvasion',
+        },
+        {
+          title: _(msg`Other`),
+          reason: 'tools.ozone.report.defs#reasonRuleOther',
         },
-        ...common,
       ],
-      starterPack: [
+    },
+    civicIntegrity: {
+      key: 'civicIntegrity',
+      title: _(msg`Civic Integrity`),
+      description: _(
+        msg`Electoral interference or political process violations`,
+      ),
+      sort: 7,
+      options: [
+        {
+          title: _(msg`Electoral Process Violations`),
+          reason: 'tools.ozone.report.defs#reasonCivicElectoralProcess',
+        },
+        {
+          title: _(msg`Disclosure and Transparency Violations`),
+          reason: 'tools.ozone.report.defs#reasonCivicDisclosure',
+        },
         {
-          reason: ComAtprotoModerationDefs.REASONVIOLATION,
-          title: _(msg`Name or Description Violates Community Standards`),
-          description: _(msg`Terms used violate community standards`),
+          title: _(msg`Voter Intimidation or Interference`),
+          reason: 'tools.ozone.report.defs#reasonCivicInterference',
+        },
+        {
+          title: _(msg`Election Misinformation`),
+          reason: 'tools.ozone.report.defs#reasonCivicMisinformation',
+        },
+        {
+          title: _(msg`Impersonation of Electoral Officials/Entities`),
```

Comment:
```suggestion
          title: _(msg`Impersonation of Electoral Officials or Entities`),
```
nit: Suggest `or` would read better than using `/`